packages/core/@generated/graphql.ts

```diff
@@ -629,6 +629,18 @@ export type StoreAggregateRating = {
   reviewCount: Scalars['Int']['output']
 }
 
+/** Attribute of a given product. */
```

Comment:
It was generated when I ran `yarn build`. It has also happened in another [PR](https://github.com/vtex/faststore/pull/2293#discussion_r1580927441) mine.
I don't know why these changes were removed after my PR was merged, but now it was necessary again, otherwise the build fails on [codesandbox](https://ci.codesandbox.io/status/vtex/faststore/pr/2307/builds/502980) 🤔

Reply:
I think the `codesandbox` problem can be a cache (tmp) problem. I also had this problem, which was fixed by running an empty commit.

This code prevents the version [from being published](https://github.com/vtex/faststore/actions/runs/9037562322/job/24836845618). So I think we need to remove it.